tests/unit/pages/mentor-home/MentorHome.spec.js

```diff
@@ -0,0 +1,47 @@
+import { screen, render } from '@testing-library/react'
+import { MemoryRouter } from 'react-router-dom'
+import { useSelector } from 'react-redux'
+
+import { ModalProvider } from '~/context/modal-context'
+import { ConfirmationDialogProvider } from '~/context/confirm-context'
+import MentorHome from '~/pages/mentor-home/MentorHome'
+
+jest.mock('react-redux', () => ({
+  useSelector: jest.fn()
+}))
+
+const MentorHomeWithProviders = () => (
+  <MemoryRouter>
```

Comment:
You don`t need memory router. you can use renderWithProviders util it include Memory Router and Provider

Reply:
fixed